widgets/flood/results.js

```diff
@@ -0,0 +1,44 @@
+var R = require('react');
+var D = R.DOM;
+var Header = require('../shared/header');
+var MapboxMap = require('../shared/mapboxMap');
+var config = require('../../config');
+var Description = require('./description');
+
+var Results = R.createClass({
+  displayName: 'flood-results',
+  propTypes: {
+    show: R.PropTypes.bool,
+    data: R.PropTypes.object
+  },
+
+  render: function() {
+    var data      = this.props.data;
+    var style     = this.props.show ? {} : { display: 'none' };
+    var floodData = data && data.floodRisk;
+    var isAtRisk  = data && data.isAtRisk;
+    var sectionEl;
+
+    if (!data) { return D.div(null); }
+
+    if (config.mapboxMapId && isAtRisk) {
```

Comment:
I would add a guard on `floodData`, which could be null and lead to a lot of trouble